first-30-minutes-of-the-writing-task-in-studieproeven.md

```diff
@@ -105,11 +115,13 @@ In the first 15 minutes, you have
  
 In the next 15 minutes, you have sketched a plan of the whole essay with keywords for all the four questions in the task to logically interconnected them. 
 
-## Now you are ready to start writing the first sentence!
+Now you are ready to start writing the first sentence!
+
+* * *
 
-## In my upcoming blog post, you will learn about relevant linguistic features for writing about data and the first question in particular! 
```

Comment:
I don't think this makes sense as a header. Rather just a paragraph in bold text.